src/graph.cpp

```diff
@@ -105,6 +106,12 @@ void Graph::edmondsKarp(const int &source, const int &target) {
     while (path(source, target)) {
         augmentPath(source, target, findBottleneck(source, target));
     }
+
+    for(Edge* edge : vertexSet[target]->getIncoming()){
```

Comment:
Once again, you shouldn't use vertexSet[target]

Reply:
I don't remember being the one that changed this, but I'll correct it nonetheless